src/entry.js

```diff
@@ -21,7 +21,7 @@ const render = () => {
   )
 }
 
-if (window && window.localStorage) window.localStorage.clear()
+// if (window && window.localStorage) window.localStorage.clear()
```

Comment:
We should figure out a way to do this better because we accidentally commit this a lot. I wonder if refactoring the way we do the HTTP requests and localStorage stuff to leverage Service Workers instead.

If we want to do something with what we currently have, I think we can check to see if the env is NOT production, then don't clear localStorage. But I want to keep the shipped app clearing local storage so that we don't run into any bugs where its just badly cached data. Thoughts?